### PR TITLE
(PE-15721) Add razor config and docs

### PR DESCRIPTION
**Pull request by cprice404**
This commit adds some docs about our razor configuration, as well
as some automation for the parts of it that can't be managed
by Puppet in the Ops modules.

### CODE REVIEW

File: jenkins-integration/razor/README.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+if you need to use an operating system that Razor doesn't support out of the box,
+or if you need to customize the default ones that come with Razor.
+
+For the tasks we're using, the tasks are just represented as files on disk.  There
+is nothing that we need to put into the Razor database for these.
+
+### Task for the `driver` node
+
+None at this time.  We're just using the standard Cent7 task that ships with razor
+and it seems so far.
```
Comment (jpinsonault) [inline jenkins-integration/razor/README.md]: I think you a  word

**Comment by jpinsonault** [comment]
This looks good to me. Thanks for writing up these docs, they're very helpful

File: jenkins-integration/razor/README.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+There are four concepts in Razor that are relevant here:
+
+* brokers
+* tasks
+* tags
+* policies
+
+The brokers and tasks are managed via the Ops puppet modules (see
+`site/profile/manifests/razor.pp`).  The files in this directory can be used to
```
Comment (rlinehan) [inline jenkins-integration/razor/README.md]: Could this be linked to?

Comment (cprice404) [inline jenkins-integration/razor/README.md]: At the time of this writing, that stuff hadn't been merged into their production branch.  I think maybe it has been now, so if you think it's worthwhile then I can check and link.

Comment (rlinehan) [inline jenkins-integration/razor/README.md]: I don't feel strongly about it? I could see it being useful, but also it seems like something that could become out of date very fast, so I don't care that much either way.

**Comment by rlinehan** [comment]
👍

**Pull request merged**